app/controllers/miq_ae_customization_controller/custom_buttons.rb

```diff
@@ -114,7 +114,7 @@ def ab_get_node_info(node)
               when "Host"
                 _("Host / Node")
               when "EmsCluster"
-                ui_lookup(:ems_cluster_types => "cluster")
+                _("Cluster / Deployment Role")
```

Comment:
@Fryguy I see what you're saying here, but as things are, this wouldn't work.

The result of `title_for_cluster()` and `title_for_host()` would then few lines down the road be used
as a key to a hash, which has its keys / values derived from `CustomButton.button_classes`, i.e.
the hash would not contain keys for all the strings these two routines would return.

At this point, I'm trying to get rid of ui_lookup_* and replace it with gettext wherever possible.